tests/models/whisper/test_modeling_whisper.py

```diff
@@ -2334,6 +2343,9 @@ def test_tiny_token_timestamp_batch_generation(self):
 
     @slow
     def test_tiny_token_timestamp_generation_longform(self):
+        # modified here since we add eos token + start tokens in generate
+        # nevertheless would have to be tested and compared with expected from openai
```

Comment:
What's the status on this comment? Should we remove it?

Reply:
It has indeed to be removed, good catch !

Reply:
applied in commit [2127991](https://github.com/huggingface/transformers/pull/34111/commits/2127991640963ea2efabcff1ca5d76cea1987cbd)